frontend/src/components/samples/SamplesListContent.js

```diff
@@ -120,7 +120,6 @@ const getTableColumns = (containersByID, individualsByID, projectsByID, sampleKi
     {
       title: "QC Flag",
       dataIndex: "qc_flag",
-      sorter: true,
```

Comment:
I would keep the sorter in case the user want to group up his multiple selection (if there was only 2 groups it would be useless to have both but for 3 groups it could be used).